src/AzureDevOpsDemoBuilder/wwwroot/Scripts/AppScripts/create.js

```diff
@@ -489,11 +489,11 @@ $('#btnSubmit').click(function () {
     if (gitHubFork === true) {
         forkGitHub = true;
         gitRepoName = $('#txtRepoName').val();
-        if (gitRepoName === "") {
-            $('#txtRepoName_Error').text("Please provide repository name");
-            $("#txtRepoName_Error").removeClass("d-none").addClass("d-block");
-            return false;
-        }
+        //if (gitRepoName === "") {
```

Comment:
Again, dead code.  I understand commenting out, but dead code can increase payload size unless it is stripped during bundling.

Reply:
removed dead code from script